table/block_based/block_based_table_reader.cc

```diff
@@ -1875,12 +1877,43 @@ CachableEntry<FilterBlockReader> BlockBasedTable::GetFilter(
                    no_io, get_context, lookup_context, prefix_extractor);
 }
 
+void BlockBasedTable::FillBlockCacheAccessRecord(
+    const BlockCacheLookupContext& lookup_context,
+    BlockCacheTraceRecord* record) const {
+  assert(record);
+  assert(rep_->ioptions.env);
+  record->access_timestamp = rep_->ioptions.env->NowMicros();
+  record->block_type = lookup_context.block_type;
+  record->block_size = lookup_context.block_size;
+  record->cf_id = rep_->table_properties
+                      ? rep_->table_properties->column_family_id
+                      : rocksdb::TablePropertiesCollectorFactory::Context::
+                            kUnknownColumnFamily;
+  record->level = rep_->level >= 0 ? rep_->level : UINT32_MAX;
+  record->sst_fd_number =
+      rep_->file ? TableFileNameToNumber(rep_->file->file_name()) : UINT64_MAX;
+  record->caller = lookup_context.caller;
+  record->is_cache_hit =
+      lookup_context.is_cache_hit ? Boolean::kTrue : Boolean::kFalse;
+  record->no_insert =
+      lookup_context.no_insert ? Boolean::kTrue : Boolean::kFalse;
+}
+
+void BlockBasedTable::FillBlockCacheAccessRecord(
+    const BlockCacheLookupContext& lookup_context, bool is_referenced_key_exist,
```

Comment:
does_referenced_key_exist ? Note that this affects multiple places in the code.

Reply:
Sorry for the typo. I renamed it to does_referenced_key_exist.